test/physics/em/KleinNishinaInteractor.test.cc

```diff
@@ -31,11 +32,14 @@ class KleinNishinaInteractorTest : public celeritas_test::InteractorHostTestBase
     void SetUp() override
     {
         using celeritas::ParticleDef;
-        using celeritas::units::speed_of_light_sq;
+        using namespace celeritas::constants;
+        using namespace celeritas::units;
 
         Base::set_particle_params(
             {{{"electron", pdg::electron()},
-              {0.5109989461, -1, ParticleDef::stable_decay_constant()}},
+              {0.5109989461 * mev_csq,
```

Comment:
Will there be a PDG lookup for things like particle mass, spin, etc.?

Reply:
Currently the PDG->ParticleDefId->ParticleDef exposes mass, decay time, and charge. None of the EM processes uses spin so that’s not in there yet.